content/doc/book/blueocean/dashboard.adoc

```diff
@@ -19,105 +19,78 @@ endif::[]
 
 = Dashboard
 
-Blue Ocean's "Dashboard" is the default view shown when you open Blue Ocean and
-shows an overview of all Pipeline projects configured on a Jenkins instance.
+Blue Ocean's *Dashboard* is the default view shown when opening Blue Ocean.
+It disaplays an overview of all Pipeline projects configured on a Jenkins instance.
 
 include::doc/book/blueocean/_blue-ocean-status.adoc[]
 
-The Dashboard consists of a blue link:#navigation-bar[navigation bar] at the
-top, the link:#pipelines-list[Pipelines list], as well as the
-link:#favorites-list[Favorites list].
+The Dashboard consists of a blue link:#navigation-bar[navigation bar] at the top, the link:#pipelines-list[Pipelines list], and the link:#favorites-list[Favorites list].
 
 [.boxshadow]
-image:blueocean/dashboard/overview.png[Overview of the
-Dashboard,role=center,width=100%]
+image:blueocean/dashboard/overview.png[Overview of the Dashboard,role=center,width=100%]
 
 == Navigation bar
 
-The Dashboard includes the blue-colored
-link:../getting-started#navigation-bar[navigation bar] along the top of the
-interface.
+The Dashboard includes the blue link:../getting-started#navigation-bar[navigation bar] along the top of the interface.
 
-This bar is divided into two sections - a common section along the top and a
-contextual section below. The contextual section changes depending on the
-current Blue Ocean page you are viewing.
+This bar is divided into two sections:
 
-When viewing the Dashboard, the navigation bar's contextual section includes
-the:
+* A common section along the top 
+* A contextual section below
+** The contextual section changes depending on the Blue Ocean page you are viewing.
 
-* *Search pipelines* field to filter the link:#pipelines-list[Pipelines list]
-  to show items containing the text you enter into this field.
-* *New Pipeline* button, which begins the
-  link:../creating-pipelines[create a Pipeline] process.
+When viewing the Dashboard, the navigation bar's contextual section includes:
 
+* *Search pipelines*: This field allows users to filter the link:#pipelines-list[Pipelines list] to match the text you enter into this field.
+* *New Pipeline*: This option begins the process of link:../creating-pipelines[creating a Pipeline].
 
 == Pipelines list
 
-The "Pipelines" list is the Dashboard's default list, and upon accessing Blue
-Ocean for the first time, this is the only list shown on the Dashboard.
+The *Pipelines* list is the Dashboard's default list, and upon accessing Blue Ocean for the first time, the only list shown on the Dashboard.
 
-This list shows the overall state of each Pipeline configured on the Jenkins
-instance (which can also include other Jenkins items). For a given item in this
-list, the following information is indicated:
+This list shows the overall state of each Pipeline configured on the Jenkins instance.
```

Comment:
Is the word "configured" required? And should it be "on" or "in"? I am not sure.

Reply:
changed to in, I believe that is more correct in this case